.goreleaser.yml

```diff
@@ -50,10 +50,39 @@ changelog:
 
 dockers:
   - dockerfile: build/Dockerfile
+    use: buildx
+    goos: linux
+    goarch: amd64
+    build_flag_templates:
+      - "--platform=linux/amd64"
     image_templates:
-      - "getwoke/woke:latest"
-      - "getwoke/woke:{{ .Major }}.{{ .Minor }}"
-      - "getwoke/woke:{{ .RawVersion }}"
+      - "getwoke/woke:latest-amd64"
+      - "getwoke/woke:{{ .Major }}.{{ .Minor }}-amd64"
+      - "getwoke/woke:{{ .RawVersion }}-amd64"
+  - dockerfile: build/Dockerfile
+    use: buildx
+    goos: linux
+    goarch: arm64
+    build_flag_templates:
+      - "--platform=linux/arm64"
+    image_templates:
+      - "getwoke/woke:latest-arm64"
+      - "getwoke/woke:{{ .Major }}.{{ .Minor }}-arm64"
+      - "getwoke/woke:{{ .RawVersion }}-arm64"
+
+docker_manifests:
```

Comment:
@caitlinelfring to be honest, me neither, but it's an option that has been there for a long time and if they remove it will be in favor in something better and we can update that in the future, there's another way to do it without goreleaser, but I wanted to keep it in the repo way, using goreleaser. 

I'll join the discord and ask about this feature, the worst thing that could happen and that they delete it soon :D I'll get back to you with more information!

Reply:
@caitlinelfring what do you think about my proposal of using the actions instead ?

Reply:
Handling this without goreleaser, but still using `docker manifest` has the same "[experimental](https://docs.docker.com/engine/reference/commandline/manifest/)" risk so I'd prefer to keep it in goreleaser. Thanks!